### PR TITLE
add SWAN RUCIO integration proposal

### PR DESCRIPTION
**Pull request by arisfkiaras**
Adding a project in collaboration with the SWAN and Rucio teams, for the integration of Rucio into SWAN.

https://indico.cern.ch/event/834069/contributions/3585129/attachments/1924599/3184747/Rucio__SWAN.pdf

### CODE REVIEW

File: _gsocproposals/2020/proposal_SWAN_RUCIO_integration.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+A working, and easy-to-use extension, installable both in SWAN and vanilla JupyterLab, which will allow users to easily stage data available on a Rucio server for interactive analysis.
+
+## Desirable Skills
+* Knowledge of JavaScript and Python
+* Knowledge of Linux and GIT
+* Experience with Jupyter Notebooks
+* Understanding of REST APIs
+
+## Mentors
```
Comment (agheata) [inline _gsocproposals/2020/proposal_SWAN_RUCIO_integration.md]: We do not impose a limit on the number of mentors usually, but 8 people for 1 student is a bit extreme... Are all these people going to take care of her/him or just may be contacts for technical advising? Normally we need to know the main mentor plus one, max. 2 people as backup.

Comment (agheata) [reply _gsocproposals/2020/proposal_SWAN_RUCIO_integration.md]: The mentors that will finally remain in this proposal should be present also in gsoc/2020/mentors.md in last name order

File: _gsocproposals/2020/proposal_SWAN_RUCIO_integration.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+title: Making Exabytes of LHC data seamlessly accessible on Jupyter Notebooks
+layout: gsoc_proposal
+project: SWAN/Rucio
```
[the remark this answers is not in the document]
Comment (arisfkiaras) [reply _gsocproposals/2020/proposal_SWAN_RUCIO_integration.md]: Decided to use SWAN as the project.
The required files are coming from https://github.com/HSF/hsf.github.io/pull/694 but if you wish I could also add them on this request.

File: _gsocproposals/2020/proposal_SWAN_RUCIO_integration.md (hunk 1 above)
Comment (arisfkiaras) [reply _gsocproposals/2020/proposal_SWAN_RUCIO_integration.md]: Removed some of the mentors.

So this gsoc project is a collaboration between the SWAN and Rucio team, so technical experts will be needed by both sides for advice and code review.
Also the project is of great importance for the ESCAPE project, so the main guidance, reports and so on will be coming from people from that project.

Comment (arisfkiaras) [reply _gsocproposals/2020/proposal_SWAN_RUCIO_integration.md]: The names of 
* Enrico Bocchi(mailto:enrico.bocchi@cern.ch)
* Diogo Castro(mailto:diogo.castro@cern.ch)
will already appear on the mentors list thanks to https://github.com/HSF/hsf.github.io/pull/694

Comment (agheata) [reply _gsocproposals/2020/proposal_SWAN_RUCIO_integration.md]: OK, we will merge #694 first

File: _gsocproposals/2020/proposal_SWAN_RUCIO_integration.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (agheata) [reply _gsocproposals/2020/proposal_SWAN_RUCIO_integration.md]: No need

**Pull request merged**